examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -268,6 +268,10 @@ trainer:
   strategy:
     _target_: lightning.pytorch.strategies.DDPStrategy
     gradient_as_bucket_view: true
+    # To enable FSDP2 (PyTorch 2.6+), replace the 'strategy' contents with the following:
+    # _target_: lightning.pytorch.strategies.ModelParallelStrategy
```

Comment:
Does it also work with `lightning.pytorch.strategies.FSDPStrategy`

Reply:
I did not try, but it's not recommended - that would be FSDP1. We should be using FSDP2.